android/assets/jsons/translations/template.properties

```diff
@@ -1176,7 +1175,7 @@ Number of your cities\ndemanding this resource for\n'We Love The King Day' =
 
 [victoryType] Victory = 
 Built [building] = 
-Add all spaceship parts in capital = 
+Add all [param] in capital = 
```

Comment:
You sure about this? It doesn't look like you changed any usages of this, and also "Add all [Spaceship part] in capital" is bad English.

Reply:
My intention was to make this string the same as the one in `VictoryTypes.json`

https://github.com/yairm210/Unciv/blob/bc9a42e452618552377c0e63a2b828b2b9cf5e7c/android/assets/jsons/Civ%20V%20-%20Gods%20%26%20Kings/VictoryTypes.json#L5-L7

but looking at it again, it seems like this victory is semi-hardcoded (as in changing whatever is inside of the [] would not matter) so the more correct approach would probably be editing `VictoryTypes.json`.